msal/src/Microsoft.Identity.Client/Internal/PlatformPlugin.cs

```diff
@@ -46,7 +43,7 @@ static PlatformPlugin()
 
         private static void InitializeWebFactoryAndPlatform()
         {
-#if !FACADE
+#if !NETSTANDARD1_1
```

Comment:
>NETSTANDARD1_1 [](start = 5, length = 14)

we are not ready to remove netstandard11?

Reply:
@henrik-me - @jennyf19 did this in parallel, the next iteration of this PR will be rebased on top of her changes.